### ISSUE TITLE
[flake] Mac ios app dismiss dialogs failing on flutter-2.13-candidate.0 CI

### ISSUE THREAD

**Issue by CaseyHillers** [flutter/flutter#107941]
Failing on https://ci.chromium.org/p/flutter/g/stable_flutter/console with the exception

```
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
```

## Examples

https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_ios%20stable%20backdrop_filter_perf_ios__timeline_summary/42/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_ios%20stable%20complex_layout_ios__compile/44/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_ios%20stable%20complex_layout_scroll_perf_ios__timeline_summary/45/overviewhttps://ci.chromium.org/ui/p/flutter/builders/prod/Mac_ios%20stable%20cubic_bezier_perf_ios_sksl_warmup__timeline_summary/32/overview
https://ci.chromium.org/ui/p/flutter/builders/prod/Mac_ios%20stable%20external_ui_integration_test_ios/42/overview

**Comment by CaseyHillers** [comment]
tl;dr from internal chat:

This was fixed on the newer version of macos, and can't really be backported. Since this is a release branch, I'm going to mark this as unplanned.
